CorePluginReporterView/src/au/gov/asd/tac/constellation/views/pluginreporter/panes/PluginReportPane.java

```diff
@@ -252,61 +251,70 @@ private void update() {
                 sequencePane.getStyleClass().add("running");
-                pluginNameLabel.getStyleClass().add(JavafxStyleManager.LIGHT_NAME_TEXT);
-                messageLabel.getStyleClass().add(JavafxStyleManager.LIGHT_MESSAGE_TEXT);
+                pluginNameLabel.getStyleClass().add(DEFAULT_NAME_TEXT);
+                messageLabel.getStyleClass().add(DEFAULT_MESSAGE_TEXT);
 
-            // If the plugin has finished
+                // If the plugin has finished
             } else {
                 sequencePane.getStyleClass().add("finished");
-                pluginNameLabel.getStyleClass().add(JavafxStyleManager.LIGHT_NAME_TEXT);
-                messageLabel.getStyleClass().add(JavafxStyleManager.LIGHT_MESSAGE_TEXT);
+                pluginNameLabel.getStyleClass().add(DEFAULT_NAME_TEXT);
+                messageLabel.getStyleClass().add(DEFAULT_MESSAGE_TEXT);
             }
-                
+
             messageLabel.setVisible(pluginReport.getLastMessage() != null);
         } else {
 
             // If the plugin has been cancelled
             if (error instanceof InterruptedException) {
                 sequencePane.getStyleClass().add("interrupted");
-                pluginNameLabel.getStyleClass().add(JavafxStyleManager.LIGHT_NAME_TEXT);
-                messageLabel.getStyleClass().add(JavafxStyleManager.LIGHT_MESSAGE_TEXT);
+                pluginNameLabel.getStyleClass().add(DEFAULT_NAME_TEXT);
+                messageLabel.getStyleClass().add(DEFAULT_MESSAGE_TEXT);
+                messageLabel.setText("Cancelled");
 
-            // If the plugin failed in an expected way
+                // If the plugin failed in an expected way
             } else if (error instanceof PluginException) {
                 sequencePane.getStyleClass().add("failed");
-                pluginNameLabel.getStyleClass().add("darkNameText");
-                messageLabel.getStyleClass().add("darkMessageText");
+                pluginNameLabel.getStyleClass().add(ERROR_NAME_TEXT);
+                messageLabel.getStyleClass().add(ERROR_MESSAGE_TEXT);
 
-            // If the plugin failed in an unexpected way
+                // If the plugin failed in an unexpected way
             } else {
                 sequencePane.getStyleClass().add("errored");
-                pluginNameLabel.getStyleClass().add(JavafxStyleManager.LIGHT_NAME_TEXT);
-                messageLabel.getStyleClass().add(JavafxStyleManager.LIGHT_MESSAGE_TEXT);
+                pluginNameLabel.getStyleClass().add(DEFAULT_NAME_TEXT);
+                messageLabel.getStyleClass().add(DEFAULT_MESSAGE_TEXT);
+
+                Writer errorWriter = new CharArrayWriter();
+                try (PrintWriter out = new PrintWriter(errorWriter)) {
+                    out.append(error.getMessage());
+                    out.append("\n\n");
+                    error.printStackTrace(out);
+                }
+                messageLabel.setText(errorWriter.toString());
```

Comment:
I might need some help understanding why the text is suddenly being changed here (when not in other places)